share/spice/coupons/coupons.js

```diff
@@ -0,0 +1,88 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_coupons = function(api_result){
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error) {
+            return Spice.failed('coupons');
+        }
+
+        var tabname, translation_ValidUntil, translation_Unknown;
+        switch (api_result.locale) {
+            case 'DE':
+                tabname = 'Gutscheine';
```

Comment:
We'd like to use generic "topics" for our categories, ideally from the list in our docs, or viewable here (on the right): https://duck.co/ia -- I think it would be appropriate to set the tab names as `Answer` because this IA directly answers queries about shopping. 

Our frontend will handle translating the word Answer into whichever local the user has set, so we should'nt need to keep a list of all the different words in the code